packages/hardhat-core/.gitignore

```diff
@@ -107,3 +107,5 @@ typings/
 .dynamodb/
 
 test/internal/hardhat-network/provider/.hardhat_node_test_cache/
+
+!.eslintrc.js
```

Comment:
Is this really needed?

Reply:
Also, our custom ignore rules should be above the "Below is Github's node gitignore template" comment

Reply:
Yeah, it's needed because of the `/*.js` ignore before.

Agree with having our rules above the comment, fixed.

Reply:
ohh! `/*.js`, I see.